content/en/docs/instrumentation/java/automatic/withspan.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Use instrumentation annotations
+description: OpenTelemetry documentaion for using instrumentation annotations with Java agent.
+weight: 2
+---
+
+For most users, the out-of-the-box instrumentation is completely sufficient and nothing more has to
+be done.  Sometimes, however, users wish to create spans for their own custom code without doing too much code change.
+
+# Dependencies
```

Comment:
Headings should start at level 1. You'll need to demote all heading in this page.